### PR TITLE
Apply needed pboProject settings in make.py via Windows Registry

### PR DESCRIPTION
**Pull request by Dahlgren**
**When merged this pull request will:**
- Set the required exclude from pbo settings as defined in https://ace3mod.com/wiki/development/setting-up-the-development-environment.html while running make.py

### CODE REVIEW

File: tools/make.py [hunk 1]
```diff
@@ -264,6 +263,17 @@ def find_depbo_tools():
 
     return requiredToolPaths
 
+def pboproject_settings():
+    """Use registry entries to configure needed pboproject settings."""
+    value_exclude = "thumbs.db,*.txt,*.h,*.dep,*.cpp,*.bak,*.png,*.log,*.pew,source,*.tga"
```
Comment (PabstMirror) [inline tools/make.py:268]: I seem to remember needing to remove *.h, but I don't think it's for ace, probably fine as it matches our wiki

File: tools/make.py [hunk 2]
```diff
@@ -264,6 +263,17 @@ def find_depbo_tools():
...
+
+    try:
+        k = mikero_windows_registry(r"pboProject\Settings", access=winreg.KEY_SET_VALUE)
+        winreg.SetValueEx(k, "m_exclude", 0, winreg.REG_SZ, value_exclude)
+        winreg.SetValueEx(k, "m_exclude2", 0, winreg.REG_SZ, value_exclude)
```
Comment (PabstMirror) [inline tools/make.py:273]: `m_exclude2` seems to be the `-B` exclusions, which should be the only way we build, but it's probably safest to set both

**Review by PabstMirror — APPROVED** [review]

File: tools/make.py (hunk 1 above)
Comment (Dahlgren) [reply tools/make.py:268]: I should probably try this build and see if that causes any issues 🤔

Comment (jonpas) [reply tools/make.py:268]: We don't use `.h` anywhere, always `.hpp`. :)

Comment (PabstMirror) [reply tools/make.py:268]: `my/config.cpp: #includes \a3\editor_f\Data\Scripts\dikCodes.h, but is excluded from pbo`

was the error pboProject gives for my addon, but this doesn't effect ace
And it shouldn't even be an error....

**Review by jonpas — APPROVED** [review]

**Pull request merged**